taipy/core/data/data_node.py

```diff
@@ -60,9 +90,10 @@ class DataNode(_Entity, _Labeled):
         last_edit_date (datetime): The date and time of the last modification.
         edits (List[Edit^]): The list of Edits (an alias for dict) containing metadata about each
             data edition including but not limited to timestamp, comments, job_id:
-            timestamp: The time instant of the writing
-            comments: Representation of a free text to explain or comment on a data change
-            job_id: Only populated when the data node is written by a task execution and corresponds to the job's id.
+                timestamp: The time instant of the writing
+                comments: Representation of a free text to explain or comment on a data change
+                job_id: Only populated when the data node is written by a task execution and
+                    corresponds to the job's id.
```

Comment:
```suggestion
                - timestamp: The time instant of the writing
                - comments: Representation of a free text to explain or comment on a data change
                - job_id: Only populated when the data node is written by a task execution and
                    corresponds to the job's id.
```

I'm not sure if this would help the list represents correctly as a bullet list when built, but I think a bullet list is better